doc/spring-petclinic-demo.md

```diff
@@ -0,0 +1,86 @@
+# Spring Petclinic Demo
+
+You need Java 1.8+ and easeagent:
+
+If you don't already have easeagent, get it and set $EASE_AGENT_PATH: [EaseAgent](../README.md#get-and-set-environment-variable)
+
+### Step 1
+Building the demo application.
+```
+$ git clone https://github.com/akwei/spring-petclinic-microservices.git
+$ cd spring-petclinic-microservices
+$ mvn -DskipTests=true package
+```
+
+export EASE_AGENT_PATH=/Users/beyond/IdeaProjects/easeagent_dir/easeagent/build/target
```

Comment:
```suggestion
Setup the `EASE_AGENT_PATH` environment variable
```shell
 $ export EASE_AGENT_PATH=/Users/beyond/IdeaProjects/easeagent_dir/easeagent/build/target
```
```

Reply:
> 

Sorry, this is a line I added by mistake when testing